src/uPlot.js

```diff
@@ -759,6 +759,7 @@ export default function uPlot(opts, data, then) {
 			s.stroke = fnOrSelf(s.stroke || hexBlack);
 			s.fill   = fnOrSelf(s.fill || null);
 			s._stroke = s._fill = s._paths = null;
+			s._focus = true;
```

Comment:
hmm, this gets the job done but i'm not excited by the semantics :(

there are additional uses for indicating explicit focus of one or multiple series besides color modification.

what do you think about making this a 3-state property: `null`, `true`, `false`. when nothing is explicitly focused it will be `null`. but when one or more series are explicitly focused, they will be `true` and the rest will be `false`.

Reply:
Yeah, i was thinking about that but was not sure how exactly to display 3 possible states. I'll update this.